Podfile

```diff
@@ -5,9 +5,9 @@ platform :ios, '7.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 
-pod 'toxcore', :podspec => 'https://raw.githubusercontent.com/Antidote-for-Tox/toxcore/0.0.0-63a82-new-av-3/toxcore.podspec'
+pod 'toxcore', :podspec => 'https://raw.githubusercontent.com/Antidote-for-Tox/toxcore/0.0.1-542338a/toxcore.podspec'
 pod 'BlocksKit', '~> 2.2.5'
 pod 'CocoaLumberjack', '~> 1.9.2'
 pod 'Masonry', '~> 0.6.1'
-pod 'objcTox', :git => 'https://github.com/Antidote-for-Tox/objcTox.git', :branch => 'audio'
+pod 'objcTox', :git => 'https://github.com/Antidote-for-Tox/objcTox.git', :branch => 'master'
```

Comment:
I think that there is no need for this customization now. We can simply use `pod 'objcTox', '0.4.0'`.
Also no need in specifying toxcore version.